### ISSUE TITLE
[20211212] Weekly AI ArXiv 만담 - 33회차

### ISSUE THREAD

**Issue by jungwoo-ha** [jungwoo-ha/WeeklyArxivTalk#33]
- News
  - [AI양재허브 2021 AI 콘서트](https://www.ai-yangjae.kr/board/board_basic/board_detail.asp?scrID=0000000170&pageNum=4&subNum=1&ssubNum=1&page=1&bd_num=00000001702021000090&act=view&s_string=)
    - 12월 15일 오후 1시 SETEC (학여울역)
    - 로열모 창시자 + AI로봇 2,3호기의 AI Insight talk
    - AI스타텁 이야기, 오픈이노베이션 대담, AI양재허브 자문위원들의 간단한 기술 트렌드 소개
  - [서울대 모두를 위한 AI 강연](https://tv.naver.com/aiis)
  - 광주 AICON 2021: http://www.aitimes.com/news/articleView.html?idxno=141905
  - AI도 특허발명자? 7개국 특허청 AI발명자 컨퍼런스: https://www.hankyung.com/it/article/2021120976891
  - WS on ImageNet PPF @ NeurIPS 2021: 12월 13일 오후 8시 ~ 밤샘 웍샵!
- Arxiv
  - Deepmind LM 연구 3부작
    - [Scaling Language Models: Methods, Analysis, and Insights from Training Gopher](https://storage.googleapis.com/deepmind-media/research/language-research/Training%20Gopher.pdf)
      - Deepmind에서 공개한 최대 280B 모델 Gopher (118페이지이므로 나눠서 보시길~), 100/124/152 태스크에서 SOTA
      - GPT2와 같은 autoregressive model. LayerNorm --> RMSNorm 대체, Relative position encoding 사용
      - SentencePiece tokenizer + Byte-level backoff for OOV
      - 300B token 학습에 활용 + 2048 window size --> GPT3와 같은 세팅, 큰모델은 메모리 이슈로 activation 과 parameter에 bfloat16 사용, Optimizer state를 위해서는 float32 쓰기를 권장
      - Appendix C에 있는 Lesson learned 가 학습하시는 분들께 아주 쏠쏠한 정보를 주는듯
      - 효율적인 훈련/인퍼런스를 위해 compression, pruning, reverse distill, sparse training 같은 걸 다 해봤는데 크게 재미못봤다.
      - 본문에는 SOTA외에 Toxicity 와 bias와 관련된 (심지어 dialog-prompt도) 분석을 많이함. 나머지 상세내용은 Appendix에서
      - 기타: https://ko.wikipedia.org/wiki/%EA%B3%A0%ED%8D%BC_(%ED%94%84%EB%A1%9C%ED%86%A0%EC%BD%9C)
![image](https://user-images.githubusercontent.com/11782739/145700400-b3dc519a-c36b-4290-b51a-c4e45bd91069.png)
![image](https://user-images.githubusercontent.com/11782739/145711686-a8aa7a7d-32da-48fb-9973-123b01eb71b6.png)
![image](https://user-images.githubusercontent.com/11782739/145700433-a6939d95-f127-4689-811f-652086faa09e.png)
![image](https://user-images.githubusercontent.com/11782739/145700629-d65fce88-4d95-436b-bd4a-f0c132315bed.png)
![image](https://user-images.githubusercontent.com/11782739/145700731-e8d9d111-0529-4233-a6af-3ce86a0a68ea.png)
![image](https://user-images.githubusercontent.com/11782739/145700739-62a67c0d-11f2-4308-a478-d0450b069310.png)

    - [Ethical and social risks of harm from Language Models](https://storage.googleapis.com/deepmind-media/research/language-research/Ethical%20and%20social%20risks.pdf)
      - Gopher와 함께 나온 LM의 윤리적/사회적 리스크에 대한 고려 사항 보고서
      - 무려 리더특성에 따른 친절한 가이드도 있음.
![image](https://user-images.githubusercontent.com/11782739/145700788-c565172c-ea14-4df8-ac92-ab8c99f819dc.png)
![image](https://user-images.githubusercontent.com/11782739/145700842-3d336a4c-5013-47b6-9754-3ebf267c0cb0.png)

    - [Improving language models by retrieving from trillions of tokens](https://storage.googleapis.com/deepmind-media/research/language-research/Improving%20language%20models%20by%20retrieving.pdf)
      - Retrieval + LM 으로 효율을 높이는 방법
      - 기존 RAG 스타일에서 시도하지 않았던 Retrieval 대상 DB의 token을 엄청 늘리고 LM 크기도 늘려봤다. Retrieval DB를 화끈하게 늘려야 성능이 확 오르는 듯
![image](https://user-images.githubusercontent.com/11782739/145701279-0d1c5280-b5d8-485f-8dd2-9e253e47ee05.png)
![image](https://user-images.githubusercontent.com/11782739/145701287-46e3a4db-e481-47ce-8f26-2d4946c27d9f.png)

  - [HairCLIP: Design Your Hair by Text and Reference Image](https://arxiv.org/abs/2112.05142v1)
    - CLIP 을 활용하여 레퍼런스 이미지나 텍스트로 헤어스타일을 집중적으로 변환하는 모델
    - 헤어칼라 interpolation 가능, 이미지 + 텍스트 description 가능, 스타일과 컬러별도 입력도 가능
    - 역시 사람 얼굴을 그대로 유지하는 것은 여전히 숙제.. 어려운 얼굴각도도 숙제
    - https://github.com/wty-ustc/HairCLIP
![image](https://user-images.githubusercontent.com/11782739/145701835-2fc3d37f-f492-4fa7-bb87-57ec5562cb1a.png)
![image](https://user-images.githubusercontent.com/11782739/145701887-708d5ab4-36c2-4054-9fca-9027f137599f.png)
![image](https://user-images.githubusercontent.com/11782739/145701898-9bb97a4d-1eb1-45f9-8614-30f286332a3e.png)
    
  - [Multimodal Conditional Image Synthesis with Product-of-Experts GANs](https://arxiv.org/abs/2112.05130v1)
    - NVidia Ming-Yu Liu 그룹의 새로운 I2I 모델 (아마도 GauGAN2 를 위한 모델?)
    - 텍스트, segmentation map, 스케치 그리고 이들의 조합으로 실사 이미지를 생성
    - https://deepimagination.cc/PoE-GAN/
![image](https://user-images.githubusercontent.com/11782739/145701960-3713a565-7006-44cc-b09c-613bbac64208.png)
![image](https://user-images.githubusercontent.com/11782739/145701985-3de8f2c1-7deb-4f79-b751-4a2e8175f2d4.png)
![image](https://user-images.githubusercontent.com/11782739/145701999-49f29c44-5028-4ace-a56c-c5131599d8aa.png)
![image](https://user-images.githubusercontent.com/11782739/145702011-af363eb7-4f33-47d5-a05e-bf66de0a5e86.png)

**Comment by hollobit** [comment]
### "내가 윤석열로 보이니"… 'AI 정치인' 선거 혁신인가, 민주주의 재앙인가
https://www.hankookilbo.com/News/Read/A2021120816460003987
- 후보 아바타를 활용한 선거운동, 정치 윤리에 부합하나
- 사실과 거짓 혼용된 정보 유통, 가짜뉴스 악용 그 책임은?
- 미국과 유럽은 선거 운동에 딥페이크 활용 엄격 제한
- 유럽연합 인공지능법 초안의 핵심은 '신뢰'

### 첨단 조작 기술(deepfake), 인공지능용 학습데이터 분야의 국면 전환 요소로 급부상
https://www.korea.kr/news/pressReleaseView.do?newsId=156485475
https://www.kipo.go.kr/kpo/BoardApp/UnewPress1App?a=&board_id=press&cp=&pg=&npp=&catmenu=m03_05_01&sdate=&edate=&searchKey=&searchVal=&bunryu=&st=&c=1003&seq=19322
- 첨단 조작 기술 기반 AI 학습 관련 특허출원 연평균 135% 이상 증가 
- 특허청은 오는 14일(화) 13시 30분 딥페이크 기반 데이터 증강 기술에 대한 특허분석 결과를 산업계와 공유 세미나 
![deepfake](https://image.newsis.com/2021/12/09/NISI20211209_0000888824_web.jpg?rnd=20211209143115)
![deepfake1](https://image.news1.kr/system/photos/2021/12/9/5112678/article.jpg/dims/optimize)

### 美 중국 인공지능(AI) 기업에 대한 투자 금지
https://www.kdfnews.com/news/articleView.html?idxno=84108
- 미 상무부는 10일(현지 시간) 중국 인공지능(AI) 기업 ‘센스타임(상탕커지·商湯科技)'을 ‘중국 군산 복합기업 명단’에 올리겠다고 발표

### 하버드 연구진 “中, 10년 안에 AI·5G 등 핵심분야서 美 따라잡을 것”
https://biz.chosun.com/international/international_general/2021/12/10/PI3AVCZXFBFFBJ7JZR6L3FWJI4/
- 하버드대 케네디스쿨의 싱크탱크인 벨퍼센터(Belfer Center)는 최근 펴낸 '거대한 기술 경쟁: 21세기의 중국과 미국'(“Great Technological Rivalry: : China vs. the U.S. in the 21st Century)
  * https://www.belfercenter.org/publication/great-rivalry-china-vs-us-21st-century
- 중국이 10년 안으로 △AI △5G △양자정보과학 △바이오기술 △반도체 △친환경에너지 등 21세기 핵심 기초 기술 분야에서 미국을 따라잡을 것이라며 이미 안면인식, 음성인식 핀테크 등 AI의 실용 분야에서 미국을 넘어섰다고 평가
- 중국은 2025년까지 과학과 기술공학, 수학 분야의 박사 학위 취득자를 미국보다 두 배 많이 배출할 것으로 예측
- 미·중 과학기술 전쟁, 2035년에 결판 난다 - https://www.joongang.co.kr/article/24103092#home

**Comment by kimyoungdo0122** [comment]
- News
    - 흥미로웠던 AI Summit(유료 행사라서 다시보기는 제공하지 않을 것 같네요ㅠㅠ)
        - 대기업+스타트업이 함께 만드는 AI생태계
        - 다양한 AI 비즈니스 사례와 노하우
        - 기술의 발전과 더불어 필요한 제도 및 윤리 장치도 함께 발전
        - 사회 속으로 더 깊게 들어온 AI
        - 제 한줄 요약은.. 
    -

**Comment by veritas9872** [comment]
Simplifying Transformer Research with xFormers & Lightning
![image](https://user-images.githubusercontent.com/33523965/145713266-33c684c8-7244-4d5e-9a29-e83d7eac70f8.png)

https://devblog.pytorchlightning.ai/part-i-simplifying-transformer-research-with-xformers-lightning-a715737b8ad4
Meta AI Research에서 xFormer 라는 새로운 Transformer library를 오픈소스했습니다.
GitHub: https://github.com/facebookresearch/xformers
기존 여러가지 Transformer 구조들은 내부적으로 차이가 많고 GPU 최적화가 되어 있지 않았는데 새로 나온 NVIDIA Triton 및 kernel fusion 방법을 적용하여 최적화되었으면서 사용하기 단순한 라이브러리를 만들었습니다. Transformer 연구를 하시는 많은 분들께 도움이 될 것으로 생각됩니다.
![image](https://user-images.githubusercontent.com/33523965/145713268-e4e991cb-7766-4f2b-a5a4-32f087f4cc57.png)


Pushing the Frontiers of Density Functionals by Solving the Fractional Electron Problem

GitHub: https://github.com/deepmind/deepmind-research/tree/master/density_functional_approximation_dm21

DeepMind에서 양자역학의 영역까지 진출했습니다. 기존 분자에서 전자의 위치를 정확하게 판단하는 것은 근사 함수를 이용해서 계산을 했지만 새로운 데이터셋을 이용해 MSE loss를 이용해 전자의 위치를 계산하는 과정을 학습했습니다.

https://www.researchgate.net/publication/356914883_Pushing_the_frontiers_of_density_functionals_by_solving_the_fractional_electron_problem


Editing a Classifier by Rewriting Its Prediction Rules
NeurIPS Link: https://papers.nips.cc/paper/2021/hash/c46489a2d5a9a9ecfc53b17610926ddd-Abstract.html

![Screenshot (1)](https://user-images.githubusercontent.com/33523965/145713624-53fe4cd2-5176-47b5-a4bf-2609dd32cd07.png)

Madry Lab 에서 나온 NeurIPS 논문인데 classifier의 출력을 추가적인 데이터를 사용하지 않고 직접적으로 editing을 하는 방식입니다. 

![Screenshot (2)](https://user-images.githubusercontent.com/33523965/145713664-60d91ee1-a094-427a-a2a8-30e115a8b239.png)

Drivable GAN 방법에 기반해 ImageNet의 alternative class를 사용해서 background 정보와 text attack를 직접적으로 공략하는 방식입니다.

**Comment by xxxnell** [comment]
* **How Do Vision Transformers Work?** [[paper]](https://git.io/JDq4y) [[code and summary]](https://github.com/xxxnell/how-do-vits-work)  

  Vision Transformer(ViT) 및 multi-head self-attention(MSA)가 성공적인 이유는 ***long-range dependency를 모델링하기 때문이 아님***을 보입니다. 구체적으로, 이 논문은 다음을 실험적으로 입증합니다.

    1. MSA는 loss landscape을 평탄화하므로, 더 나은 일반화와 강건성을 얻을 수 있습니다. ***(Figure 1)*** long-range dependency는 오히려 loss를 non-convex하게 만들어 학습을 방해합니다.
    2. Conv와 MSA는 서로 다르게 행동합니다. 예를들어, Conv가 high-pass filter인 것과 달리 MSA는 low-pass filter입니다. 따라서, 그 둘은 상호보완적입니다 ***(Figure 2)***. 전통적인 Transformer에선 MLP block이 Conv의 역할을 하므로, MSA block과 MLP block의 결합은 합리적인 귀결입니다.
    3. MSA가 모델의 끝이 아닌 stage의 끝에 위치할 때 성능이 가장 두드러지게 개선됩니다 ***(Figure 3)***. 이는 multi-stage 모델은 여러 개별적인 모델을 선형 결합한 것처럼 행동하기 때문입니다.

  이 발견들을 모두 고려함으로써 MSA와 Conv를 조합하는 단순한 디자인 규칙을 제시합니다. 이 규칙을 따르는 새롭게 제안된 모델은 전통적인 ViT와 달리, ImageNet은 물론 심지어 CIFAR와 같은 작은 데이터에서도 CNN의 성능을 상회합니다. 


<p align="center">
<img width="85%" alt="loss-landscape" src="https://user-images.githubusercontent.com/930317/145713458-2058c214-fc6d-457f-8f0b-c90ba80950a9.png">
</p>

<p align="center">
<img width="85%" alt="fourier" src="https://user-images.githubusercontent.com/930317/145713465-1b3a6b85-80e7-4ac8-be0b-30ba75fd7ff8.png">
</p>

<p align="center">
<img width="85%" alt="architecture" src="https://user-images.githubusercontent.com/930317/145713468-752fa114-ee67-4465-8371-537d643e6691.png">
</p>

**Comment by nick-jhlee** [comment]
CLeaR: Causal Learning and Reasoning
URL: https://www.cclear.cc/2022
Causal learning등 지속적인 학습에 초점을 맞춘 새로운 학회가 생겼습니다. 이미 deadline은 끝났습니다만, 앞으로 커질 가능성이 많은 학회인 것 같습니다. (물론 causal의 c도 모릅니다...)

tentative conference 기간: Apr 11 - Apr 13, 2022

**Comment by jwlee-ml** [comment]
[NL-Augmenter: A Framework for Task-Sensitive Natural Language Augmentation](https://arxiv.org/abs/2112.02721v1)

여러 학교, 회사 소속 100명 가까운 저자
NLP의 data augmentation을 해주는 python framework
신청곡(X)논문(O)입니다. 다음주에 누군가 해주시면 감사히 듣겠습니다 ㅎㅎ

![image](https://user-images.githubusercontent.com/25281219/145715781-cf10eea4-dd4f-4384-87e7-70e230cb9b89.png)
